unitTests/ui/MissingTermsTest.ts

```diff
@@ -0,0 +1,217 @@
+import * as Mock from '../MockEnvironment';
+import { MissingTerms } from '../../src/ui/MissingTerm/MissingTerms';
+import { $$ } from '../../src/Core';
+import { FakeResults } from '../Fake';
+
+export function MissingTermsTest() {
+  describe('MissingTerm', () => {
+    const mockComponent = (query: string, option = {}) => {
+      return Mock.advancedResultComponentSetup<MissingTerms>(
+        MissingTerms,
+        fakeResult,
+        new Mock.AdvancedComponentSetupOptions(null, option, (env: Mock.MockEnvironmentBuilder): Mock.MockEnvironmentBuilder => {
+          env.withLiveQueryStateModel().queryStateModel.set('q', query);
+          return env;
+        })
+      );
+    };
+
+    let test: Mock.IBasicComponentSetup<MissingTerms>;
+    let fakeResult = FakeResults.createFakeResult();
```

Comment:
Erhm, you declare this after the function inside which you are using it (`mockComponent`), which is kinda weird.

Also, the `fakeResult` is reused between each test and not reset each time.
Should be created inside a `beforeEach` block.